bug(ci): Local development build job failing.

https://github.com/monetr/monetr/blob/5071e5a100325284f695a4b7d808ac12aa2cd66e/docker-compose.yaml#L40-L45

https://github.com/monetr/monetr/actions/runs/3054245452/jobs/4925942607#step:7:1810

The monetr service is taking a bit too much time to become healthy inside the CI pipeline, this is probably because everything has to be pulled from scratch every time as well as the health check was recently adjusted to make local development on real computers a bit faster to startup [See #1023](https://github.com/monetr/monetr/commit/636e71b724b3267e7b1cd15d977f933b405ba9ae#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9).

This is causing the healthcheck to fail inside CI:

```
Container monetr-redis-1  Waiting
Container monetr-ui-1  Waiting
Container monetr-monetr-1  Waiting
Container monetr-kms-1  Waiting
Container monetr-postgres-1  Waiting
Container monetr-documentation-1  Waiting
Container monetr-nginx-1  Waiting
Container monetr-mail-1  Waiting
Container monetr-kms-1  Healthy
Container monetr-documentation-1  Healthy
Container monetr-mail-1  Healthy
Container monetr-postgres-1  Healthy
Container monetr-redis-1  Healthy
Container monetr-ui-1  Healthy
Container monetr-nginx-1  Healthy
container for service "monetr" is unhealthy
make: *** [Makefile:296: develop] Error 1
```

We should simply revert the health check back to what it was.

--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -39,10 +39,10 @@ services:
     working_dir: /build
     healthcheck:
       test: [ "CMD", "curl", "-f", "http://localhost:4000/api/health" ]
-      interval: 5s
+      interval: 10s
       timeout: 10s
       retries: 24
-      start_period: 30s
+      start_period: 60s
     environment:
       air_wd: /build
       DISABLE_GO_RELOAD: ${DISABLE_GO_RELOAD:-false}

--- a/pkg/ui/filesystem.go
+++ b/pkg/ui/filesystem.go
@@ -1,5 +1,3 @@
-//go:build !noui
-
 package ui
 
 import (

--- a/pkg/ui/no_ui.go
+++ b/pkg/ui/no_ui.go
@@ -1,0 +1,10 @@
+//go:build noui
+
+package ui
+
+import (
+	"embed"
+)
+
+// When we are building with `noui` then do nothing here.
+var builtUi embed.FS

